qubesmanager/settings.py

```diff
@@ -1684,6 +1686,22 @@ def template_apps_change(self):
                 self.app_list_manager.has_missing
             )
 
+    def filter_apps(self):
```

Comment:
the problem with this approach is that if someone searches for "xfce  terminal" or "terminal xfce" they won't find either. I think we should find things matching every word in the search query?